packages/protocol/contracts/stability/Exchange.sol

```diff
@@ -54,6 +57,9 @@ contract Exchange is
   uint256 public updateFrequency;
   uint256 public minimumReports;
 
+  uint256 public minSupplyForStableBucketCap;
+  FixidityLib.Fraction public stableBucketMaxFraction;
+
   bytes32 public stableTokenRegistryId;
```

Comment:
```suggestion
  bytes32 public stableTokenRegistryId;
  
  uint256 public minSupplyForStableBucketCap;
  FixidityLib.Fraction public stableBucketMaxFraction;
```

In the proxy pattern that we use, new variables must always be added at the bottom. https://blog.openzeppelin.com/proxy-patterns/

Reply:
@martinvol thanks